lib/OpenCloud/ObjectStore/Resource/DataObject.php

```diff
@@ -293,6 +300,26 @@ public function getEtag()
     {
         return $this->etag ? : $this->content->getContentMd5();
     }
+    
+    /**
+     * @param $manifest
```

Comment:
Could you please add a description for this parameter? We are starting to auto-generate API documentation so the description would be useful to readers.